### DISCUSSION TITLE
I can't get my address in user profile

### DISCUSSION THREAD

**Discussion by sentance** [swellstores/community#504] [Help] [answered]
Hello,

We are looking to enhance our user experience by implementing a new feature for managing addresses within our shop. As part of this initiative, we've integrated a GraphQL query into our codebase to retrieve address data
https://developers.swell.is/frontend-api/address-management#list-addresses 

However, despite implementing all the necessary functions to send requests, we are encountering an issue where the server consistently returns an empty array, indicating a lack of response.

We would greatly appreciate your assistance in resolving this matter. Any insights, guidance, or support you can provide to help us debug and rectify this issue would be invaluable.

Thank you for your attention to this matter.

**Comment by logeshswell** [comment]
1. Can you share your store URL?
2. Can you share a screenshot from graphQL playground to help you with debugging the issue?
  * https://<store_id>.swell.store/playground
3. Do you pass `swell-session` in the headers?
  * This value should be retrieved from the [user login](https://developers.swell.is/frontend-api/session-management#log-in).
  * You can also validate the `accountId` value present in swell-session: https://developers.swell.is/frontend-api/session-management#verify-session-token

**Comment by sentance** [comment]
1. Store url https://www.flon.co.uk
2. 
![Screenshot_16](https://github.com/swellstores/community/assets/10882664/db5b64f3-afb0-4541-8f69-62e4b253679a)
3. Yes.
Test user has an address in his account but returning value is null

**Reply by logeshswell** [reply, marked as answer]
Everything looks right at your end. We'll take a deeper look at this issue and get back to you if I get some updates.

**Comment by sentance** [comment]
Any update?

**Reply by flonUK** [reply]
@logeshswell , any updates? It's a live site and users need to manage their payments and addresses. This should've been in the Horizon theme. It's been over a week since we first asked, not ideal. If there is a major issue could you let us know a time frame or something, it is not a good look having to explain to customers that they cannot add or edit their addresses. Hope you look into this please. thanks

**Reply by logeshswell** [reply]
This issue is prioritized and yet to be worked on. I also bumped up the priority level to get it released ASAP.
I will keep you posted if there's any new update.

**Reply by logeshswell** [reply]
This issue is resolved. Please check and confirm if it's working fine.
